tidy3d/components/data/sim_data.py

```diff
@@ -298,17 +443,52 @@ def plot_field(  # pylint:disable=too-many-arguments, too-many-locals, too-many-
 
         # get the DataArray corresponding to the monitor_name and field_name
 
-        # intensity
+        # deprecated intensity
         if field_name == "int":
-            field_data = self.get_intensity(field_monitor_name)
-            val = "abs"
+            log.warning(
```

Comment:
I like that you did this in a way that shouldn't break any existing notebooks or tests. But I think we'll still want to update several notebooks to avoid this warning, at the time of merging this PR?

Reply:
Will do!